tests/index.test.js

```diff
@@ -418,7 +418,7 @@ describe('ServerlessWebpack', () => {
                 });
             });
             it('should skip compiling when requested', () => {
-              slsw.skipCompile = true;
+              slsw.skipCompile = false;
```

Comment:
Something doesn't add up here, did we flip the boolean meaning in `validate.js`?

Reply:
I've just reverted what you did here: https://github.com/serverless-heaven/serverless-webpack/commit/44dc73e124358bdb2308963454498517289a12ee#diff-59e25b254be93038f106111be580b6fb54c6963b6c4e7ef744e58fb8f2b3e3a2

Reply:
In fact, it changes nothing if it's true or false because the value is updated by the new option. I'll just remove the line.